src/Modal/ModalDialog.jsx

```diff
@@ -117,6 +119,7 @@ ModalDialog.propTypes = {
    */
   isBlocking: PropTypes.bool,
   zIndex: PropTypes.number,
+  isOverflowVisible: PropTypes.bool,
```

Comment:
For all the properties of this component, we provide a short description for the convenience of consumers. I suggest doing this also for `zIndex` and `isOverflowVisible`

For example:
```
  /** Specifies the z-index of the modal */
  zIndex: PropTypes.number,
  /** Specifies whether overflow is visible in the modal */
  isOverflowVisible: PropTypes.bool,
```

Reply:
Good catch! Addressed here https://github.com/openedx/paragon/compare/2f7f60c6a9ac8348c614ae9954ac9fd7a8a6c293..fc31ad7e3c3c69968d89712a15f9d42672ddabd6